### ISSUE TITLE
Invalid FB data len

### ISSUE THREAD

**Issue by wangxiuwen** [yangh/sevensquare#25]
1540796807605 Invalid FB data len: 9322496 require 9331200

**Comment by wangxiuwen** [comment]
on Nut pro2

**Comment by elsandosgrande** [comment]
I got "Invalid FB data len: 3620864 require 8294400" before a segfault. _Wonders if the phone and laptop specifications are necessary alongside a list of installed packages_

**Comment by SailingDreams** [comment]
I also get "Invalid FB data len" before a device disconnected and segmentation fault. I am getting this on a Snapdragon 820 EVM. EVM is connected to a AMD Threadripper running Ubuntu 18

I tried both the "-s and -p" options from the ubuntu command line and get the output
Screencap on device options -q -s false false

I was able to get it working without the "Invalid FB data Len" by compiling and running the CMakeLists.txt below after running the "make && sudo make install". I have to run sevensquare from within CLion. For some reason running the same compiled app in a terminator window causes the same "Invalid FB data Len" error.

1560807374681 CAP FB in 960 ms 
1560807374713 Invalid FB data len: 9207808 require 9216000 
1560807375741 CAP FB in 924 ms 
1560807375772 Invalid FB data len: 9207808 require 9216000 
1560807376790 CAP FB in 913 ms 
**1560807376823 Device disconnected** 
1560807376923 ADB Wait for device 
1560807376932 ADB Found 
1560807376938 Compressed data transfer true 
1560807377425 Screencap on device options -q -s false false 
1560807378357 CAP FB in 932 ms 
1560807378391 Invalid FB data len: 9207808 require 9216000 
1560807378645 Input device info 0 "qpnp_pon" 3 "3" 
1560807378645 Found key input "qpnp_pon" 0 
1560807378698 Found POWER key define in "qpnp_pon" 116 0 
1560807378814 Input device info 1 "HDMI CEC User or Deck Control" 3 "3" 
1560807378814 Found key input "HDMI CEC User or Deck Control" 1 
1560807378990 Input device info 12 "Logitech USB Receiver" 1179667 "120013" 
1560807378990 Found key input "Logitech USB Receiver" 12 
1560807379185 Input device info 13 "Logitech USB Receiver" 31 "1f" 
1560807379185 Found touch panel "Logitech USB Receiver" 13 
1560807379303 Input device info 2 "qbt1000_key_input" 11 "b" 
1560807379303 Found touch panel "qbt1000_key_input" 2 
1560807379437 Input device info 4 "hbtp_vm" 7 "7" 
1560807379438 Found unknown input "hbtp_vm" 4 
1560807379461 CAP FB in 1061 ms 
1560807379474 Invalid FB data len: 1572864 require 9216000 
Segmentation fault (core dumped)

CMakeLists.txt
============
cmake_minimum_required(VERSION 3.14)
project(sevensquare)

find_package(Qt4 4.4.3 REQUIRED QtGui QtXml)

set(CMAKE_CXX_STANDARD 14)

include_directories(src)

add_executable(sevensquare
        build/moc_adbfb.cpp
        build/moc_cubescene.cpp
        build/qrc_seven-square.cpp
        src/adbfb.cpp
        src/adbfb.h
        src/cubecellitem.cpp
        src/cubecellitem.h
        src/cubescene.cpp
        src/cubescene.h
        src/debug.h
        src/fbcellitem.cpp
        src/fbcellitem.h
        src/input-event-types.h
        src/keycodes.h
        src/keymap-generated.h
        src/keymap.h
        src/main.cpp
        src/utils.cpp
        src/utils.h)

target_link_libraries(sevensquare Qt4::QtGui Qt4::QtXml)
